scss/_colors.scss

```diff
@@ -3,3 +3,6 @@ $white: #FFF;
 $black-lighter-5: rgba(0,0,0,0.5);
```

Comment:
you can also use [hexadecimal color value](https://alligator.io/css/hex-code-colors-alpha-values/) with the alpha value.